src/Surfnet/StepupGateway/GatewayBundle/Controller/GatewayController.php

```diff
@@ -220,7 +220,9 @@ public function renderSamlResponse(
         $httpResponse = $this->render($view, $parameters);
 
         $ssoCookieService = $this->get('gateway.service.sso_2fa_cookie');
```

Comment:
Consider moving this line into the if clause.

Reply:
Consider DI for gateway.service.sso_2fa_cookie

Reply:
Good suggestions guys. @pablothedude I'm following your suggestion.
@parijke your suggestion will be part of the SF upgrade party! Don't you worry